docs/website/versioned_docs/version-3.0.0/command-reference/delete.md

```diff
@@ -1,40 +1,42 @@
 ---
-title: odo delete
+title: odo delete component
 sidebar_position: 4
 ---
 
-`odo delete` command is useful for deleting resources that are managed by odo.
+`odo delete component` command is useful for deleting resources that are managed by `odo`. It deletes the component and its related innerloop and outerloop resources from the cluster.
 
-## Deleting a component
-
-To delete a _devfile_ component, you can execute `odo delete`.
+There are 2 ways to delete a component:
+1. [Delete with access to Devfile](#delete-with-access-to-devfile)
+2. [Delete without access to Devfile](#delete-without-access-to-devfile)
 
+## Delete with access to Devfile
 ```shell
-odo delete
+odo delete component
 ```
+`odo` analyzes the Devfile _present in the current directory_ for the component and the outerloop resources.
```

Comment:
Why are we emphasizing `present in the current directory`? Such emphasis makes sense in a blog, but do we really need them in docs? 

I believe that it's a very subjective thing and different people could have different views about this. But as far as docs are concerned, we should try to use a uniform language and approach. If we want to highlight something, we should first ask ourselves if there's an admonition that could be used. If not, we must define what different levels of emphasis mean. In the sense, what should be put in bold, what should be italicized, what should be underlined, or when should we use a combination of two or all of these.

This is more of an open conversation than a review to change something. I'm simply sharing my thoughts.

Reply:
Okay, that makes sense.
So, the reason I emphasized it here is because the section title says `Delete with access to devfile`, this could mean to 2 things - 1) Devfile is accessible because it is present in the current directory, 2) Devfile is accessible through something like `--context`.
Even though odo no longer supports --context, given it's history, I thought it made sense to emphasize the first part about it being present in the current directory.
I am not sure if  adding admonition here would make sense because I would like to highlight it in the sentence itself.